src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -678,6 +685,15 @@ export default class QueryStringInputUI extends Component<Props, State> {
               >
                 {this.getQueryString()}
               </EuiTextArea>
+              {this.props.iconType ? (
+                <div className="euiFormControlLayoutIcons">
+                  <EuiIcon
```

Comment:
@snide I wonder if we don't want the icon to be everywhere, rather than configurable.
I don't think anyone would really want to configure this.